File: POO/temas/22/1/rb/2.rb
frutas = ["piña", "coco", "higo", "dátil", "kiwi"]

encontrado = false
i = 0
while i < frutas.length && !encontrado
    if frutas[i].length == 5
        encontrado = true
    else
        i = i + 1
    end
end

puts "encontrado: #{encontrado}"
puts "i = #{i}"
puts frutas[i]

File: POO/temas/22/5/py/2.py
frutas = ["piña", "coco", "higo", "dátil", "kiwi"]

encontrado = False
i = 0
while i < len(frutas) and not encontrado:
    if len(frutas[i]) == 5:
        encontrado = True
    else:
        i = i + 1

print(f"encontrado: {encontrado}")
print(f"i = {i}")
print(frutas[i])
